README.md

```diff
@@ -63,6 +63,52 @@ higher-level APIs:
 
 ## Guides
 
+### Quickstart
+
+1. Clone this repo:
+   ```
+   git clone https://github.com/kubernetes-sigs/kubefed.git
+   ```
+1. Start a [kind](https://kind.sigs.k8s.io/) cluster:
+   ```
+   kind create cluster
+   ```
+1. Deploy kubefed:
+   ```
+   make deploy.kind
+   ```
+
+You now have a Kubernetes cluster with kubefed up and running. The cluster has been joined to itself and you can test federation of resources like this:
+
+1. Verify the `KubeFedCluster` exists and is ready:
+   ```
+   kubectl -n kube-federation-system get kubefedcluster
+   ```
+   **If you're on macOS** the cluster will not immediately show as ready. You need to [change the API endpoint's URL first](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/cluster-registration.md#joining-kind-clusters-on-macos):
+   ```
+   ./scripts/fix-joined-kind-clusters.sh
+   ```
+1. Create a namespace to be federated:
+   ```
+   kubectl create ns federate-me
+   ```
+1. Tell kubefed to federate that namespace (and the resources in it):
+   ```
+   ./bin/kubefedctl federate ns federate-me
+   ```
```

Comment:
You could check if the namespace was propagated.

Reply:
Also, we are not mentioning anything about placement definitions or explaining them.

Reply:
> Also, we are not mentioning anything about placement definitions or explaining them.

Given the goal of this guide being providing as little steps as possible to try out kubefed I don't see a need to mention this here. The more advanced docs are explaining that pretty well already.